components/sections/docs/docs-mobile.nav.tsx

```diff
@@ -0,0 +1,207 @@
+/** @jsx jsx */
+
+import { jsx } from "theme-ui";
+import { useState } from "react";
+import {
+  MobileMenuIcon,
+  SearchIcon,
+  CloseMobileNav,
+} from "components/svgs/darkMode";
+import MobileLogo from "./mobile-logo";
+import DarkModeSwitch from "components/primitives/darkModeSwitch";
+import DocsMenu from "./docs-menu";
+
+type Props = {
+  setColorMode: React.Dispatch<React.SetStateAction<string>>;
+  colorMode: string;
+  handleClose: React.MouseEventHandler<HTMLDivElement>;
+  selected: string;
+};
+
+const OpenMenu = ({
+  colorMode,
+  setColorMode,
+  handleClose,
+  selected,
+}: Props) => {
+  return (
+    <div
+      sx={{
+        height: "100vh",
+        position: "sticky",
+        top: "0",
+        overflowY: "scroll",
+        backgroundColor: "docs.background",
+        width: "100%",
+        display: "flex",
+        flexDirection: "column",
+        padding: "32px 24px",
+      }}>
+      <div
+        sx={{
+          width: "100%",
+          display: "flex",
+          justifyContent: "flex-end",
+          position: "sticky",
+          top: "0",
+        }}
+        onClick={handleClose}>
+        <CloseMobileNav color={colorMode === "dark" ? "#fff" : "#131418"} />
+      </div>
+      <DocsMenu selected={selected} />
+      <div sx={{ position: "fixed", bottom: "100px", right: "16px" }}>
+        <DarkModeSwitch colorMode={colorMode} setColorMode={setColorMode} />
+      </div>
+    </div>
+  );
+};
+
+const DocsMobileNav = ({ setColorMode, colorMode, selected }) => {
+  const [menuOpen, setMenuOpen] = useState<boolean>(false);
```

Comment:
when the menu opens, document scroll must be disabled. See [this](https://github.com/basementstudio/lettermatic/blob/master/src/components/sections/header.tsx#L68-L103) for reference